docs/core/windows-prerequisites.md

```diff
@@ -118,11 +118,14 @@ To develop .NET Core 1.x apps in Visual Studio, [download and install Visual Stu
 > For more information about the project format changes, see [High-level overview of changes](./tools/cli-msbuild-architecture.md).
 ---
 
+<a name="vs-mapping"></a>
+
 > [!TIP]
 > To verify your Visual Studio 2017 version:
 >
 > * On the **Help** menu, choose **About Microsoft Visual Studio**.
 > * In the **About Microsoft Visual Studio** dialog, verify the version number.
+>   * For .NET Core 2.2 Preview 1 apps, Visual Studio 2017 version 15.9 Preview 1 or higher.
```

Comment:
Question: would it be clearer to phrase this as "Visual Studio 2017 version 15.9 (currently in Preview)" to clarify that it's in the Preview channel?